src/schema/article-renderer.tsx

```diff
@@ -226,8 +209,14 @@ function renderElement(props: RenderElementProps): React.ReactNode {
     return <StyledP>{children}</StyledP>
   }
   if (element.type === 'h') {
-    const Comp = StyledHx[element.level]
-    return <Comp id={element.id}>{children}</Comp>
+    return React.createElement(
+      `h${element.level}`,
+      {
+        className: `serlo-h${element.level}`,
```

Comment:
This is a bit problematic together with purging: https://tailwindcss.com/docs/optimizing-for-production#writing-purgeable-html

Adding an object like `{1: 'serlo-h1', 2: 'serlo-h2', ...}` should do the trick.